infra/image/system-service/fixipaip.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -eu
+
+HOSTNAME=$(hostname)
+IP=$(hostname -I | cut -d " " -f 1)
+
```

Comment:
should we double check if we have hostname properly configured inside the container?

if [ -z "${HOSTNAME}" ] || [ -z "${IP}" ]; then
    echo "ERROR: Failed to retrieve hostname or IP address"
    exit 1
fi